docs/source/data/kedro_dvc_versioning.md

```diff
@@ -0,0 +1,294 @@
+# Data and pipeline versioning with Kedro and DVC
+
+This document explains how to use [DVC](https://dvc.org/) to version datasets and pipelines in your Kedro project. DVC is a tool to develop reproducible machine learning projects. It can be installed on Visual Studio Code, any system terminal, and used as a Python library.
+
+This tutorial assumes you have experience with the Git CLI and Kedro CLI commands but does not require any prior knowledge of DVC.
+
+## Versioning data with .dvc files
+
+### Initialising the repository
+
+For this example, we will be using a Kedro `spaceflights-pandas` starter project, which includes pre-configured datasets and pipelines. To create this starter project locally, use the command:
+
+`kedro new --starter=spaceflights-pandas --name=space-dvc`
+
+For more information about starter projects, see the [Kedro starters documentation](https://docs.kedro.org/en/stable/starters/starters.html) page.
+
+To use DVC as a Python library, install it using `pip` or `conda`, for example:
+`pip install dvc`
+
+Since DVC works alongside Git to track data changes, initialise the Kedro project as a git repository: `git init`.
+
+Then, initialise DVC in the project: `dvc init`. This will create the `.dvc` directory inside the project.
+
```

Comment:
nit: Would like to see these steps in code blocks so the users don't miss the steps at a quick glance and can copy paste